ado_wrapper/resources/runs.py

```diff
@@ -3,7 +3,8 @@
 from datetime import datetime
 from typing import TYPE_CHECKING, Any, Literal, TypedDict
 
-from ado_wrapper.resources.builds import Build
+# from ado_wrapper.resources.builds import Build
+from wrapper.builds import Build, BuildDefinition
```

Comment:
Also please change this line:
from wrapper.builds import Build, BuildDefinition
to
from ado_wrapper.resources.builds import Build, BuildDefinition